package.json

```diff
@@ -44,6 +49,7 @@
     "@testing-library/jest-dom": "^5.12.0",
     "@testing-library/react": "^11.2.6",
     "@testing-library/user-event": "^13.1.8",
+    "@types/i18next-fs-backend": "^1.0.0",
```

Comment:
Can be removed

Reply:
i18next-fs-backend is used in the tests to load the translation.json files.